pkg/cloud/services/securitygroup/securitygroups.go

```diff
@@ -434,7 +434,7 @@ func (s *Service) revokeAllSecurityGroupIngressRules(id string) error {
 			}
 			if _, err := s.EC2Client.RevokeSecurityGroupIngress(revokeInput); err != nil {
 				record.Warnf(s.scope.InfraCluster(), "FailedRevokeSecurityGroupIngressRules", "Failed to revoke all security group ingress rules for SecurityGroup %q: %v", *sg.GroupId, err)
-				return errors.Wrapf(err, "failed to revoke security group %q ingress rules", id)
+				return err
```

Comment:
Same comment as above

Reply:
Similar reason is applicable here also what I mentioned above